hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -5157,6 +5176,20 @@ private String reconfOzoneKeyDeletingLimitPerTask(String newVal) {
     return newVal;
   }
 
+  private String reconfKeyLifecycleServiceEnabled(String newVal) {
+    boolean enabled = StringUtils.isEmpty(newVal) ?
```

Comment:
Can we throw exception if newVal is null?  I think "null" is an invalid input, so it's better response to user with correct info. Will the message in exception be passed to client and show as the output of CLI?